### PR TITLE
Add sealing key support

### PR DESCRIPTION
**Pull request by L0czek**
This add support for reading key material used in sealing keys derivation.

### CODE REVIEW

**Comment by L0czek** [comment]
The modified rsi kernel modules canbe found at https://github.com/islet-project/realm-manager/tree/dev/m.szaknis/key_sealing/realm/linux-rsi

File: lib/rust-rsi/src/ioctl/kernel.rs [hunk 1]
```diff
@@ -65,6 +66,27 @@ impl RsiAttestation
...
+
+#[repr(C)]
+pub struct RsiSealingKey
+{
+    pub(super) flags: u64,
+    pub(super) svn: u64,
+    pub(super) realm_sealing_key: [u8; 32]
+}
+
+impl RsiSealingKey {
```
Comment (Havner) [inline lib/rust-rsi/src/ioctl/kernel.rs]: !!!

File: lib/rust-rsi/src/ioctl/kernel.rs [hunk 2]
```diff
@@ -7,12 +7,13 @@
...
 mod internal
 {
-    use super::{RsiMeasurement, RsiAttestation};
+    use super::{RsiMeasurement, RsiAttestation, RsiSealingKey};
 
     nix::ioctl_read!(abi_version, b'x', 190u8, u64);
     nix::ioctl_readwrite_buf!(measurement_read, b'x', 192u8, RsiMeasurement);
     nix::ioctl_write_buf!(measurement_extend, b'x', 193u8, RsiMeasurement);
     nix::ioctl_readwrite_buf!(attestation_token, b'x', 194u8, RsiAttestation);
+    nix::ioctl_readwrite_buf!(sealing_key, b'x', 195u8, RsiSealingKey);
```
Comment (Havner) [inline lib/rust-rsi/src/ioctl/kernel.rs]: Those numbers were take from somewhere, SMCC? They were not random, hence 191 is unused here. Maybe don't use 195, but something higher? Might be wise to find where did I take it from ;-)

File: tools/rsictl/src/subcmds.rs [hunk 3]
```diff
@@ -154,3 +154,34 @@ pub(crate) fn verify_platform(args: &VerifyPlatformArgs) -> GenericResult
     tools::verify_print_platform(&token, &key)?;
     Ok(())
 }
+
+#[derive(ValueEnum, Debug, Copy, Clone)]
+pub(crate) enum SealingKeyFlags {
```
Comment (Havner) [inline tools/rsictl/src/subcmds.rs]: !!!

File: tools/rsictl/src/tools.rs [hunk 4]
```diff
@@ -49,3 +52,26 @@ pub(crate) fn verify_print_platform(token: &[u8], key: &[u8]) -> Result<(), rust
     rust_rsi::print_token_platform(&token);
     Ok(())
 }
+
+impl From<&SealingKeyFlags> for u64 {
```
Comment (Havner) [inline tools/rsictl/src/tools.rs]: !!!

**Review by Havner — APPROVED** [review]

**Pull request merged**